com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -952,29 +941,7 @@ void ResetData()
         /// <seealso cref="IActionReceiver.OnActionReceived"/>
         public virtual void Heuristic(in ActionBuffers actionsOut)
         {
-            // Disable deprecation warnings so we can call the legacy overload.
```

Comment:
There was a
```
            Debug.LogWarning("Heuristic method called but not implemented. Returning placeholder actions.");
```
message in the deprecated `float[]` code. I think we should keep that here.